### PR TITLE
checked notes  and made some changes

### PR DESCRIPTION
**Pull request by mira-gruenwald**
for P0132 I had copied the wrong name from the old file. I changed it according to the project bibliography.
I added the corresponding texts from the bibliog. also for P0136 and P0138.

added source where reading is recorded see #93

### CODE REVIEW

File: data/Act.csv [hunk 1]
```diff
@@ -71,7 +71,7 @@ A00069,P0003,,T00081,,1961,1964?,,T00045,38,,2017-09-27,MG,2017-09-27,MG,,,,
 A00070,P0068,,T00081,,1961,1964?,,T00045,38,,2017-09-27,MG,2017-09-27,MG,,,,
 A00071,P0132,AT11?,T00082,,1975,1979,The author published some translated chapters of Charlie Chaplin's autobiography on the magazine 'Foreign Films' which was founded by Huang Tianmin,T00043,455,,2017-09-27,MG,2017-10-02,DP,,,,
 A00072,P0109,AT01,T00083,,1970,1975,story-telling context: in her spare time Shu Ting tells this in her own words,T00014,304,,2017-09-27,MG,2017-09-27,MG,,,,
-A00073,P0132,AT13,T00084,P0167,1978,1978,Chen wrote this Poem for Mr. Yan Wenliang. The author first touched upon impressionism fine arts in Yan's home at which he was totally impressed.,T00043,452,,2017-09-29,MG,2017-10-08,MG,,,,
```
Comment (inamoto4) [inline data/Act.csv]: Please do not forget to add the date of your last update.

File: data/Person.csv [hunk 2]
```diff
@@ -243,16 +243,16 @@ P0130,Xu,Chenggang,en,,,,,,,,,,,,,,,,2017-07-28,MG,2017-09-16,CW
...
 P0134,林,莽 ,zh,male,,,,,,,,,,,,,,,2017-08-11,MG,2017-11-07,LH
 P0135,Liao,Yiwu,en,male,0,0,0,0,1958,,Lao Wei,en,PL0062,,,https://de.wikipedia.org/wiki/Liao_Yiwu,,,2017-08-11,MG,2017-11-06,OS
 P0135,廖,亦武,zh,male,0,0,0,0,1958,,,,PL0062,,,https://de.wikipedia.org/wiki/Liao_Yiwu,,,2017-08-11,MG,2017-11-06,OS
-P0136,Bao,Jia,en,,yes,1970,1972,PL0072,1954,NULL,,,PL0002,,,,,"Mira: are we sure that this is the guy we are looking for? You don‘t mean this guy, right: https:// did not find him in the text filebaike.baidu.com/item/%E6%B1%A0%E5%AE%9D%E5%98%89 Btw, I",2017-08-26,MG,2017-09-16,CW
-P0136,宝,嘉,zh,,yes,1970,1972,PL0072,1954,NULL,,,PL0002,,,,,,2017-08-26,MG,2017-09-16,CW
+P0136,Bao,Jia,en,,yes,1970,1972,PL0072,1954,NULL,,,PL0002,,,T00057,495,"Mira: are we sure that this is the guy we are looking for? You don‘t mean this guy, right: https:// did not find him in the text filebaike.baidu.com/item/%E6%B1%A0%E5%AE%9D%E5%98%89 Btw, I",2017-08-26,MG,2017-11-08,MG
+P0136,宝,嘉,zh,,yes,1970,1972,PL0072,1954,NULL,,,PL0002,,,T00057,495,This person is a reader recorded in the Act-file and in the project bibliography,2017-08-26,MG,2017-11-08,MG
```
Comment (inamoto4) [inline data/Person.csv]: Maybe I am mixing something up (it is already late here), but T0057 refers to a text by the premodern author Bai Juyi, right? How can this be the source of our modern Bao Jia?

File: data/Act.csv (hunk 1 above)
Comment (inamoto4) [reply data/Act.csv]: The whole file seems to be a bit out of sync. How can that be?

File: data/Act.csv [hunk 3]
```diff
@@ -71,7 +71,7 @@ A00069,P0003,,T00081,,1961,1964?,,T00045,38,,2017-09-27,MG,2017-09-27,MG,,,,
...
+A00073,P0132,AT13,T00084,P0167,1978,1978,Tao wrote this Poem for Mr. Yan Wenliang. The author first touched upon impressionism fine arts in Yan's home at which he was totally impressed.,T00043,452,,2017-09-29,MG,2017-11-08,MG,,,,
```
Comment (inamoto4) [inline data/Act.csv]: Please do not forget to add the date of your last update. I have changed it accordingly.

File: data/Person.csv (hunk 2 above)
Comment (mira-gruenwald) [reply data/Person.csv]: Ok I see, I got something wrong then...it's not the text he read but the text where his reading is recorded. I will  change it.

**Comment by inamoto4** [comment]
I have set the columns in the act file back in sync again. Should be all okay by now.

**Pull request merged**